app/controllers/items_controller.rb

```diff
@@ -0,0 +1,78 @@
+# frozen_string_literal: true
+
+# ItemsController
+class ItemsController < ApplicationController
+  before_action :set_item, only: %i[show edit update destroy]
+
+  # GET /items
+  # GET /items.json
+  def index
+    @items = Item.all
+  end
+
+  # GET /items/1
+  # GET /items/1.json
+  def show; end
+
+  # GET /items/new
+  def new
+    @item = Item.new
+  end
+
+  # GET /items/1/edit
+  def edit; end
+
+  # POST /items
+  # POST /items.json
+  def create
+    @item = Item.new(item_params)
+
+    respond_to do |format|
+      if @item.save
+        format.html { redirect_to @item, notice: 'Item was successfully created.' }
+        format.json { render :show, status: :created, location: @item }
+      else
+        format.html { render :new }
+        format.json { render json: @item.errors, status: :unprocessable_entity }
+      end
+    end
+  end
+
+  # PATCH/PUT /items/1
+  # PATCH/PUT /items/1.json
+  def update
+    respond_to do |format|
+      @item.image.purge if params[:image]
```

Comment:
How does it work when rolling back a transaction?
Perhaps only the file is deleted, leaving a record with no attachments. 😢